code/controllers/master.dm

```diff
@@ -7,6 +7,9 @@
  *
  **/
 
+/// This needs to be run as soon as physically possible
+GLOBAL_REAL_VAR(list/__typeinfo_cache) = new
```

Comment:
This should probably be after the debugger ;)

Reply:
WHAT IF THE DEBUGGER HAS TYPEINFO WACKA? HUH?